scss/donate.scss

```diff
@@ -1,5 +1,166 @@
 /*@use 'vars';*/
 //temporary until resolve _vars.scss issue
+
+// HERO SECTION BEGINS HERE
+// .hero-clouds {
+//   position: absolute;
+//   width: 100%;
+//   height: 100vh;
+//   z-index: 1;
+//   @media only screen and (max-width: 767px) {
```

Comment:
be sure to test mobile responsiveness, looks like the hero covers more of the sections underneath when I make the screen smaller